samcli/lib/utils/sam_logging.py

```diff
@@ -28,9 +32,12 @@ def configure_logger(logger, formatter, level):
         if handlers:
             log_stream_handler = handlers[0]
         else:
-            log_stream_handler = logging.StreamHandler()
+            log_stream_handler = (
+                RichHandler(console=Console(stderr=True), show_time=False, show_path=False, show_level=False)
```

Comment:
Can we define as many `Console` instances as possible, or should we define one and use it during the whole execution?

Reply:
we re-use once its configured AFAIK within the scope of the process, that's the existing methodology. But good to check.

Reply:
I see, but how can someone access `Console` instance from somewhere else? Let's say we wanted to add progress bars to build/package/deploy steps, where they can find the previously instantiated `Console` instance?

Reply:
At this point you can't. since the console instance is exclusive to log_stream_handler. however in a future PR we could move the console instance to thread local storage? thoughts?